tests/translate/translate_remap_profile_2d.py

```diff
@@ -90,3 +94,4 @@ def __init__(self, grid):
             "a4_4": {"serialname": "q4_4_2", "istart": 0, "iend": grid.ie - 3},
         }
         self.ignore_near_zero_errors = {"q4_4_2": True}
+        self.js = grid.js
```

Comment:
probably don't need this anymore? also, the translate objects, all have a self.grid btw, so could do self.grid.js. but this does not harm